src/implementations/twig/components/picture/README.md

```diff
@@ -0,0 +1,69 @@
+# ECL Blockquote component
```

Comment:
this is the blockquote's README :)
I would suggest to separate the prs, in any case, we are introducing a new "component", even though mostly transparent for the users, but still it feels it deserves its own enty in the changelog :)